### DISCUSSION TITLE
Supervision Integration with Other frameworks.

### DISCUSSION THREAD

**Discussion by hardikdava** [roboflow/supervision#233] [General]
Hello all, I am creating this discussion to create a list of good frameworks which supports computer vision tasks. The main idea is to integrate result format conversion to `supervision`. Once we have the final list of frameworks then we can do lot of things with it such an example would be to benchmark accuracy performance with all the possible models. 

List of frameworks:

- [x] ultralytics (included in next release #221 )
- [ ] paddlepaddle
	- [x] paddledet ( #187 )
	- [ ] paddlesegmentation
- [ ] open-mmlab
	- [x] mmdetection, mmyolo (#173)
	- [ ] mmsegmentation 
- [ ] super-gradient 
	- [x] detections
	- [ ] segmentation

The community will have the standard way to compare performance. It is observed that all the frameworks uses their own benchmarking method such as mAP for object detection. This way we all have fair comparision rather than the results from their own implementation. This can leads to standardizing the metric for comparision.

@SkalskiP @onuralpszr @kirilllzaitsev

**Comment by SkalskiP** [comment]
As for `super-gradient`, we support YOLO-NAS. Are there any other models you can run with `super-gradient`?

**Reply by hardikdava** [reply]
Yeah, check their [model-zoo](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/model_zoo.md)

**Reply by SkalskiP** [reply]
Hah. Looks like I missed something :D
